app/models/post.rb

```diff
@@ -46,4 +57,28 @@ def self.tweet_create_params_from_api(tweet)
     }
   end
 
+  # GRAMS ---------------------------------------------------------------------
+
+  def self.create_many_grams_from_api(grams)
+    # Instagram API returns most recent post first, but we want it in reverse
+    grams.reverse!
+    grams.each do |gram|
+      post = Post.create(create_gram_params_from_api(gram))
+      Medium.create(url: gram["images"]["standard_resolution"]["url"], post_id: post.id)
+    end
+  end
+
+  def self.create_gram_params_from_api(gram)
+    { uid: gram["id"],
+      body: gram["caption"]["text"],
+      post_time: convert_unix_to_datetime(gram["created_time"]),
+      prey_id: Prey.find_by(uid: gram["user"]["id"]).id,
+      url: gram["link"],
+      provider: "instagram"
+    }
+  end
+
+  def self.convert_unix_to_datetime(time)
+    Time.at(time.to_i).to_datetime
```

Comment:
Do you have to do this? I thought that Rails usually handles the conversion to datetime on its own when it saves it to the db.